include/sdf/Param.hh

```diff
@@ -406,6 +401,17 @@ namespace sdf
                                     const std::string &_valueStr,
                                     ParamVariant &_valueToSet) const;
 
+    /// \brief Method used to get the string representation from a ParamVariant
+    /// \param[in] _config Print configuration for the string output
+    /// \param[in] _typeName The data type of the value
+    /// \param[in] _value The value
+    /// \param[in] _valueStr The string representation of the value
+    public: bool SDFORMAT_VISIBLE StringFromValueImpl(
```

Comment:
We found it necessary for `ValueFromStringImpl` because it was used in a function template in this file. But I don't see `StringFromValueImpl` being used in a template so we I don't think it's necessary here.

Reply:
Thanks for spotting this! Removed in https://github.com/ignitionrobotics/sdformat/pull/729/commits/345642248eef8fafe595259ec2c86693209a6eb9